.github/workflows/docs-deploy.yml

```diff
@@ -31,6 +31,22 @@ jobs:
         run: npm ci
         working-directory: ./docs
 
+      - name: Install sub-package dependencies
```

Comment:
Since the docs sidebar generation relies on importing and parsing TypeScript source of each sub-package we wish to document, we need to pull in the dependencies of each sub-package before building can begin.